EventFilter/L1TRawToDigi/plugins/implementations_stage2/CaloLayer1Unpacker.cc

```diff
@@ -21,55 +21,64 @@ namespace l1t {
       auto ctp7_phi = block.amc().getBoardID();
       const uint32_t* ptr = block.payload().data();
 
+      int amc_slot = block.amc().getAMCNumber();
+
       int N_BX = (block.header().getFlags() >> 16) & 0xf;
       //      std::cout << " N_BX calculated " << N_BX << std::endl;
 
       int HCALFB = (block.header().getFlags() >> 15) & 0x1;
+      //int slot7card = (block.header().getFlags() >> 14) & 0x1;
```

Comment:
This does not seem to be used: perhaps a remnant of some previous implementation?
```suggestion
```